dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1580,6 +1580,10 @@ a.download-video {
     width: 800px;
   }
 
+  .finish-signup #instructions p {
+    display: inline;
```

Comment:
why is this necessary? I don't think it's desirable to make paragraphs behave as if they aren't paragraphs.

Reply:
I created a JIRA task and TODO's to resolve this next sprint